beacon-chain/rpc/validator/proposer.go

```diff
@@ -169,11 +175,13 @@ func (vs *Server) eth1Data(ctx context.Context, slot uint64) (*ethpb.Eth1Data, e
 	// Look up most recent block up to timestamp
 	blockNumber, err := vs.Eth1BlockFetcher.BlockNumberByTimestamp(ctx, eth1VotingPeriodStartTime)
 	if err != nil {
-		return nil, errors.Wrap(err, "could not get block number from timestamp")
+		log.WithError(err).Error("Failed to get block number from timestamp")
```

Comment:
We also could consider warning for these, i dont feel strongly for either, up to you!

Reply:
Its an error that should be monitored